### ISSUE TITLE
Loader does not detect optimization packages.

### ISSUE THREAD

**Issue by AndrewRademacher** [cyrilwanner/next-optimized-images#86]
When running either in dev mode or running a build the library does not appear to find image optimization libraries in spite of them being installed.

```
➜  landing-site git:(master) ✗ yarn run dev
yarn run v1.17.0
$ ts-node server.ts
next-optimized-images ❯ WARNING!
next-optimized-images ❯ No package found which can optimize images.
next-optimized-images ❯ Starting from version 2 of next-optimized-images, all optimization is optional and you can choose which ones you want to use.
next-optimized-images ❯ For help during the setup and installation, please read https://github.com/cyrilwanner/next-optimized-images#optimization-packages

next-optimized-images ❯ If you recently updated from v1 to v2, please read https://github.com/cyrilwanner/next-optimized-images/blob/master/UPGRADING.md
next-optimized-images ❯ If this is on purpose and you don't want this plugin to optimize the images, set the option `optimizeImages: false` to hide this warning.
```

Package File:

```json
{
  "scripts": {
    "dev": "ts-node server.ts",
    "build": "next build",
    "start": "ts-node server.ts",
    "storybook": "start-storybook -p 6006 -c .storybook -s .",
    "storybook-nobrowser": "start-storybook --ci -p 6006 -c .storybook -s .",
    "build-storybook": "build-storybook -c .storybook -o build-storybook; cp -R static/* build-storybook/static",
    "start-storybook": "ts-node server-storybook.ts"
  },
  "dependencies": {
    "@babel/core": "^7.4.3",
    "@fortawesome/fontawesome-svg-core": "^1.2.17",
    "@fortawesome/free-solid-svg-icons": "^5.8.1",
    "@fortawesome/react-fontawesome": "^0.1.4",
    "@storybook/addon-actions": "^5.0.10",
    "@storybook/react": "^5.0.10",
    "@storybook/theming": "^5.0.11",
    "@tryghost/content-api": "^1.2.4",
    "@types/classnames": "^2.2.7",
    "@types/express": "^4.16.0",
    "@types/isomorphic-fetch": "^0.0.35",
    "@types/next": "^8.0.3",
    "@types/node": "^11.13.5",
    "@types/react": "^16.8.14",
    "@types/react-dom": "^16.0.11",
    "@types/storybook__addon-actions": "^3.4.2",
    "@types/storybook__react": "^4.0.1",
    "@types/styled-jsx": "^2.2.8",
    "@zeit/next-typescript": "^1.1.1",
    "babel-loader": "^8.0.5",
    "babel-preset-react-app": "^7.0.2",
    "classnames": "^2.2.6",
    "express": "^4.16.4",
    "express-basic-auth": "^1.1.6",
    "fork-ts-checker-webpack-plugin": "^1.0.4",
    "glamor": "^2.20.40",
    "image-trace-loader": "^0.1.0",
    "imagemin-gifsicle": "^6.0.1",
    "imagemin-mozjpeg": "^8.0.0",
    "imagemin-optipng": "^7.0.0",
    "imagemin-svgo": "^7.0.0",
    "isomorphic-fetch": "^2.2.1",
    "jimp": "^0.6.4",
    "lqip-loader": "^2.2.0",
    "moment": "^2.24.0",
    "next": "8.1.0",
    "next-compose-plugins": "^2.1.1",
    "next-optimized-images": "^2.3.3",
    "prop-types": "^15.7.2",
    "react": "^16.7.0",
    "react-docgen-typescript": "^1.12.4",
    "react-dom": "^16.7.0",
    "react-icons": "^3.6.1",
    "react-infinite-scroller": "^1.2.4",
    "react-moment": "^0.9.2",
    "responsive-loader": "^1.2.0",
    "sharp": "^0.22.0",
    "storybook-readme": "^5.0.2",
    "styled-components": "^4.2.0",
    "styled-jsx": "3.2.1",
    "svg-sprite-loader": "^4.1.6",
    "ts-node": "^8.1.0",
    "typescript": "^3.4.4",
    "webp-loader": "^0.5.0",
    "webpack": "^4.34.0"
  },
  "peerDependencies": {
    "file-loader": "^4.0.0"
  }
}

```

Yarn: 1.17.0
Node: v12.4.0
MacOS: 10.14.5 (18F132)

**Comment by AndrewRademacher** [comment]
Additionally this issue also appears when using npm version 6.9.0.

**Comment by AndrewRademacher** [comment]
To add a clue in the puzzle, with no changes to the package manifest, the loader works as expected when building docker images of the application.  Running Alpine:3.9.

**Comment by cyrilwanner** [comment]
Hi @AndrewRademacher, thank you for the bug report! I just pushed a patch to npm which should fix this problem. Can you install `v2.5.2` and confirm that this resolved this issue? Thanks!

**Comment by AndrewRademacher** [comment]
After updating to `v2.5.2` everything seems to work except when I add responsive-loader back into the mix. Then I get the following error on MacOS

```
yarn run v1.17.0
$ ts-node server.ts
Error: Cannot find module 'responsive-loader'
Require stack:
- /Users/andrew/src/naut/landing-site/node_modules/next-optimized-images/lib/loaders/index.js
- /Users/andrew/src/naut/landing-site/node_modules/next-optimized-images/lib/index.js
- /Users/andrew/src/naut/landing-site/next.config.js
- /Users/andrew/src/naut/landing-site/node_modules/next-server/dist/server/config.js
- /Users/andrew/src/naut/landing-site/node_modules/next-server/next-config.js
- /Users/andrew/src/naut/landing-site/node_modules/next-server/dist/server/next-server.js
- /Users/andrew/src/naut/landing-site/node_modules/next/dist/server/next-dev-server.js
- /Users/andrew/src/naut/landing-site/node_modules/next/dist/server/next.js
- /Users/andrew/src/naut/landing-site/server.ts
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:623:15)
    at Function.resolve (internal/modules/cjs/helpers.js:21:19)
    at detectLoaders (/Users/andrew/src/naut/landing-site/node_modules/next-optimized-images/lib/loaders/index.js:48:26)
    at Object.webpack (/Users/andrew/src/naut/landing-site/node_modules/next-optimized-images/lib/index.js:27:31)
    at Object.getBaseWebpackConfig [as default] (/Users/andrew/src/naut/landing-site/node_modules/next/dist/build/webpack-config.js:308:32)
    at HotReloader.getWebpackConfig (/Users/andrew/src/naut/landing-site/node_modules/next/dist/server/hot-reloader.js:162:37)
    at HotReloader.start (/Users/andrew/src/naut/landing-site/node_modules/next/dist/server/hot-reloader.js:168:25)
    at DevServer.prepare (/Users/andrew/src/naut/landing-site/node_modules/next/dist/server/next-dev-server.js:76:9)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Just like last time, this issue does not occur under Alpine:3.9.

**Comment by cyrilwanner** [comment]
Oops, I've missed a require statement for the responsive-loader. I've also fixed that and just published `v2.5.3` on npm which should hopefully finally fix the issues with node 12.
Thanks again for reporting this bug!

**Comment by AndrewRademacher** [comment]
That did the trick! Optimized images now works as expected in all of my environments. Thank you again for the extremely rapid responses.
